packages/launcher/src/renderer/launcher/PermissionsView.js

```diff
@@ -224,21 +223,21 @@ export default class PermissionsView extends Component<Props, State> {
     }
 
     const { permissionsSettings } = this.state
-    let options, hasOptions
+    let options, domainOptions
 
     if (key === 'WEB_REQUEST') {
       if (Array.isArray(value)) {
         if (!value.length) {
           return null
         }
-        options = (
+        options = null
+        domainOptions = (
           <Domains key={key}>
-            {value.map(domain => (
+            {['google.com', ...value].map(domain => (
```

Comment:
Any reason we have `google.com` hardcoded here please?

Reply:
Good spot, not sure why this was added, was it intentional to commit this @diogoperillo?

Reply:
Oops... sorry.. I left it there... My bad.

Reply:
here's the PR to fix it.
https://github.com/MainframeHQ/mainframe-os/pull/357